core/src/main/java/hudson/model/Computer.java

```diff
@@ -176,6 +176,21 @@
 
     protected final Object statusChangeLock = new Object();
 
+    private transient final List<RuntimeException> terminatedBy = Collections.synchronizedList(new ArrayList
+            <RuntimeException>());
+
+    public void recordTermination() {
```

Comment:
`@since` (can just use `TODO` as the version string as a placeholder to remind the merger)